src/Smp.Web/ClientApp/src/app/messages/messages.component.scss

```diff
@@ -2,11 +2,31 @@
 @import '~sass/header-partial';
 
 .messages-body {
-  height: 100vh;
+  padding: 4rem 0 0 0;
   margin: 0;
-  padding: 0;
 
   .messages-content {
-    padding: 4rem 0 4.3rem 0;
+    height: calc(100vh - 8rem);
```

Comment:
Similarly, to the above padding, this height gets calculated based on the header and footer. Footer has been set to 4rem so make 4rem + 4rem = 8rem.